coral/src/app/router_utils.ts

```diff
@@ -27,6 +38,16 @@ enum ApprovalsTabEnum {
   CONNECTORS = "APPROVALS_TAB_ENUM_connectors",
 }
 
+const TOPIC_OVERVIEW_TAB_ID_INTO_PATH = {
+  [TopicOverviewTabEnum.OVERVIEW]: "overview",
+  [TopicOverviewTabEnum.ACLS]: "acls",
```

Comment:
I wonder if we should call have `subscriptions` in the URL to mirror the copy in the UI.